posts/2016-11-02-serverless-v1.1.0.md

```diff
@@ -0,0 +1,44 @@
+---
+title:  New Release: 1.1.0
+description:  Today we're releasing 1.1.0 of the Serverless Framework
+date:  2016-10-27
+thumbnail:  https://cloud.githubusercontent.com/assets/20538501/19769976/25c14630-9c23-11e6-9698-8cda3b8809d8.jpg
+layout:  Post
+---
+
+With the release of 1.0 a few weeks ago we're now settling into our bi-weekly release cycle and can announce our next release today with 1.1.0.
+
+We decided to go with a bi-weekly release cycle to force us to release new features and bug fixes quickly and to really focus on shipping for our community. At the same time we're investing heavily in testing, integration tests, code review and really making sure that our stability is high.
+
+## Using Semver
+
+From now on we're going to use strict Semver for Serverless. So if we jump to 2.0 or 3.0 don't be surprised.
+
+## Highlights of 1.1.0
+
+Following are a few highlights, you can check out all the details of the release [in the Changelog](https://github.com/serverless/serverless/releases/tag/v1.1.0).
+
+### Invoke functions locally
+
+You can run `serverless invoke local -f FUNCTION_NAME` to run functions on your system. You can even set event data. Take a look at the [invoke documentation](https://serverless.com/framework/docs/providers/aws/cli-reference/invoke/) for more details. This is currently only available for NodeJS functions.
+
+### Rollback
+
+You can now get a list of releases stored in your S3 deployment bucket with `serverless deploy list` and use the timestamps of those deployments for rollback with `serverless rollback -t TIMESTAMP`.
+
+### Version Pinning
+
+To make sure we can release new updates quickly and you don't accidentally deploy with a wrong version of Serverless we implemented [version pinning](https://serverless.com/framework/docs/providers/aws/guide/version/).
+
+Simply set `frameworkVersion: "=1.0.3"` in your `serverless.yml` file and Serverless will refuse any commands if the version doesn't match.
```

Comment:
"=1.0.3" -> "1.0.3"

imho less confusing. Maybe a note that it works the same as versioning in npm might help